static/style.css

```diff
@@ -0,0 +1,145 @@
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.navbar {
+  width: 100%;
+  overflow: auto;
+  justify-content: space-between;
+}
+
+.logo {
+  background-image: url("/Users/mrblue/Documents/projects/mb-portfolio/static/logo.png");
+  background-repeat: no-repeat;
+  width: 200px;
+  height: 80px;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  padding: 10px 24px;
+  justify-content: space-between;
+  background-color: white;
+  position: fixed;
+  width: 100%;
+}
+
+main {
+  overflow-y: auto;
+}
+
+.button {
+  position: absolute;
+  top: 100%;
+}
+
+.bio {
+  height: 718px;
+  width: 100%;
+  background-color: #fff;
+  background-size: contain;
+  background-position: center;
+  padding: 100px 20px;
+  background-origin: border-box;
+  border-bottom-left-radius: 30%;
+}
+
+.bio-header {
+  font-family: Playfair Display, sans-serif;
+  font-size: 50px;
+  font-weight: 400;
+  line-height: 50px;
+  letter-spacing: 0;
+  text-align: left;
+  padding: 0 39px;
+}
+
+.bio-info {
+  font-family: Poppins, sans-serif;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 21px;
+  letter-spacing: -0.01em;
+  align-items: center;
+  justify-content: center;
+  padding: 8px 45px;
+  flex-direction: row;
+  width: auto;
+}
+
+.container {
+  display: flex;
+  background-image: url("/Users/mrblue/Documents/projects/mb-portfolio/static/image2.png");
+  background-repeat: no-repeat;
+  justify-content: center;
+  flex-direction: column;
+  width: auto;
+  align-items: center;
+  object-fit: contain;
+  background-size: contain;
+  height: auto;
+  padding: 110% 59px;
+  margin: 5%;
+}
```

Comment:
- [ ] I noticed that your design has two scroll bar, this is accidentally caused by the presence of the `.container` class, Kindly remove the container class and it's styles entirely and add the background image to the `.bio` class or something close, as such.

```
.bio {
  height: 718px;
  width: 100%;
  background-color: #fff;
  background-size: contain;
  background-position: center;
  padding: 100px 20px;
  background-origin: border-box;
  border-bottom-left-radius: 30%;
  background-image: url("./image2.png");
  background-repeat: no-repeat;
  background-size: contain;
  background-position: center 30rem;
}
```
![mobile](https://user-images.githubusercontent.com/41163571/236008195-40c157fb-70f8-4a5a-af66-1b4e59f464f3.gif)